.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install Dependencies
         run: yarn
       - name: Run tests
-        run: yarn test
+        run: yarn prebuild && yarn test
```

Comment:
I'm thinking about putting it here instead of adding a `pretest` script in our package.json because I don't want these scripts running every time we want to run tests. If anyone has any thoughts about this, please let me know

Reply:
I think I could just mock it too, so just let me know what you all want me to do 🙂